common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -1,4 +1,9 @@
 {
+  "commands.wynntils.clearCaches.clickHere": "Click here to confirm.",
+  "commands.wynntils.clearCaches.deleting": "Clearing caches and closing game in 5 seconds...",
+  "commands.wynntils.clearCaches.warn": "This will clear all API and update caches, are you sure you want to continue?",
```

Comment:
```suggestion
  "commands.wynntils.clearCaches.warn": "This will clear all API and update caches, and restart Minecraft. Are you sure you want to continue?",
```

Reply:
```suggestion
  "commands.wynntils.clearCaches.warn": "This will clear all API and update caches, and close Minecraft. Are you sure you want to continue?",
```